### DISCUSSION TITLE
svtr yml用来训练，爆显存，batch_size=1也不行

### DISCUSSION THREAD

**Discussion by nissansz** [PaddlePaddle/PaddleOCR#12745] [Q&A]
#### 运行环境 / Runtime Environment
- OS:
- Paddle:
- PaddleOCR:

#### 附件 / Appendix

Out of memory error on GPU 0. Cannot allocate 120.000000MB memory on GPU 0, 7.953613GB memory has been allocated and available memory is only 47.500000MB.

Please check whether there is any other process using GPU 0.
1. If yes, please stop them, or start PaddlePaddle on another GPU.
2. If no, please decrease the batch size of your model.

 (at ..\paddle\fluid\memory\allocation\cuda_allocator.cc:87)
. (at ..\paddle\fluid\imperative\tracer.cc:307)

C:\F\pycharm2020.2\PaddleOCR-2.7.5>
C:\F\pycharm2020.2\PaddleOCR-2.7.5>

**Comment by GreatV** [comment]
原因：
其他程序或进程可能正在使用 GPU 内存。
解决方案：
使用 nvidia-smi 命令查看当前 GPU 的使用情况，并找到占用 GPU 内存的进程。
杀掉不必要的进程以释放 GPU 内存。

**Comment by nissansz** [comment]
没有其他占用，改resnet34 yml可以训练，一换svtr就不行

**Comment by nissansz** [comment]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/41010669/4ba18c01-be36-4d6a-9848-cd88fdd8e99e)

**Comment by nissansz** [comment]
./configs/rec/rec_svtrnet_ch.yml 这个配置可以训练
ch_PP-OCRv4_rec_svtr_large4lan.yml 这个配置也可以训练，这个配置有什么问题吗？看到有这个issue https://github.com/PaddlePaddle/PaddleOCR/issues/12440

./configs/rec/PP-OCRv4/ch_PP-OCRv4_rec.yml 这个配置即使设置batch size =1也爆显存，怎么修改这个配置让其能训练？

**Comment by BillEliot** [comment]
是这样的，他好像不会释放显存，会一直申请，4090显存都不够。。。
